core/gesture.ts

```diff
@@ -38,19 +38,19 @@ import type {WorkspaceSvg} from './workspace_svg.js';
 
 
 /**
- * Note: In this file "start" refers to touchstart, mousedown, and pointerstart
- * events.  "End" refers to touchend, mouseup, and pointerend events.
+ * Note: In this file "start" refers to pointerdown
+ * events.  "End" refers to pointerup events.
  */
-// TODO: Consider touchcancel/pointercancel.
+// TODO: Consider pointercancel.
```

Comment:
I'm not clear what there actually is to consider; TOUCH_MAP already listens for pointercancel if you listen for mouseup or touchcancel; I suppose we could also listen for it if you listen for pointerup?

Reply:
Sweetness, well if you don't understand I don't think there's anyone on earth who understands, so we should just remove this TODO.

Reply:
I'll spend a few minutes trying to figure out what this was

Reply:
According to MDN, pointercancel is supposed to handle cases where "the browser determines that there are unlikely to be any more pointer events, or if after the [pointerdown](https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerdown_event) event is fired, the pointer is then used to manipulate the viewport by panning, zooming, or scrolling."

Possible cases include:
"
- A hardware event occurs that cancels the pointer activities. This may include, for example, the user switching applications using an application switcher interface or the "home" button on a mobile device.
- The device's screen orientation is changed while the pointer is active.
- The browser decides that the user started pointer input accidentally. This can happen if, for example, the hardware supports palm rejection to prevent a hand resting on the display while using a stylus from accidentally triggering events.
- The [touch-action](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action) CSS property prevents the input from continuing."

It says that "Note: After the pointercancel event is fired, the browser will also send [pointerout](https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerout_event) followed by [pointerleave](https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerleave_event)."

So if we've already got pointerup, we don't need to worry about pointercancel, and we'll also already handle it directly. So yup, the TODO is obsolete.